tdrs-backend/tdpservice/scheduling/db_backup.py

```diff
@@ -103,34 +111,30 @@ def restore_database(file_name, postgres_client, database_uri=DATABASE_URI):
     with open('/home/vcap/.pgpass', 'w') as f:
         f.write(DATABASE_HOST+":"+DATABASE_PORT+":"+DATABASE_DB_NAME+":"+DATABASE_USERNAME+":"+DATABASE_PASSWORD)
     os.environ['PGPASSFILE'] = '/home/vcap/.pgpass'
-    os.system('chmod 0600 ~/.pgpass')
+    os.system('chmod 0600 /home/vcap/.pgpass')
 
     os.system(postgres_client + "pg_restore" + " -p " + DATABASE_PORT + " -h " +
               DATABASE_HOST + " -U " + DATABASE_USERNAME + " -d " + DATABASE_DB_NAME + " " + file_name)
     return True
 
 
-def upload_file(file_name, bucket, object_name=None, region='us-gov-west-1'):
-    """Upload a file to an S3 bucket."""
-    """
+def upload_file(file_name, bucket, sys_values, object_name=None, region='us-gov-west-1'):
+    """Upload a file to an S3 bucket.
+
     :param file_name: file name being uploaded to s3 bucket
     :param bucket: bucket name
     :param object_name: S3 object name. If not specified then file_name is used
     :param region: s3 AWS region to be used. defaults to government west
     :return: True is file is uploaded, False if not successful
     """
-    print("Uploaded {} to S3 in {}/{}".format(file_name, bucket, object_name))
     if object_name is None:
         object_name = os.path.basename(file_name)
-    # upload the file
-    s3_client = boto3.client('s3', region_name=region)
-    try:
-        s3_client.upload_file(file_name, bucket, object_name)
-        print("Uploaded {} to S3:{}{}".format(file_name, bucket, object_name))
-        return True
-    except Exception as e:
```

Comment:
Removed this exception because I was running into my errors being swallowed and getting cryptic error message here and elsewhere.